src/main/java/io/appium/java_client/android/geolocation/AndroidGeoLocation.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.android.geolocation;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+import static java.util.Optional.ofNullable;
+
+public class AndroidGeoLocation {
+    private Double longitude;
+    private Double latitude;
+    private Double altitude;
+    private Integer satellites;
+    private Double speed;
+
+    /**
+     * Initializes AndroidLocation instance.
+     */
+    public AndroidGeoLocation() {
+
+    }
+
+    /**
+     * Initializes AndroidLocation instance with longitude and latitude values.
+     *
+     * @param longitude longitude value
+     * @param latitude latitude value
+     */
+    public AndroidGeoLocation(double longitude, double latitude) {
```

Comment:
@mykola-mokhnach does it make sense to match **order** of parameter with org.openqa.selenium.html5.Location
```
 public Location(double latitude, double longitude, double altitude) {
    this.latitude = latitude;
    this.longitude = longitude;
    this.altitude = altitude;
  }
```
?